src/components/Vote/VoteOverview/VoteCard.js

```diff
@@ -220,7 +228,11 @@ function VoteCard({
               </div>
             ))}
           </div>
-          {isViewAll && addressNumber > 3 && (
+          {/* view all is clicked and there are still empty lines to fill up */}
+          {!isViewAll && remainingToLoad > 0 && (
+            <Icon className="loading-icon" type="loading" />
+          )}
+          {isViewAll && addressNumber > 3 && remainingToLoad > 0 && (
```

Comment:
Why 3 and not 4 though? I see it works correctly, just curious

Reply:
actually 'addressNumber > 3' is uselss now, i will delete it